src/seedu/addressbook/data/AddressBook.java

```diff
@@ -74,4 +76,8 @@ public boolean equals(Object other) {
                 || (other instanceof AddressBook // instanceof handles nulls
                         && this.allPersons.equals(((AddressBook) other).allPersons));
     }
+
```

Comment:
Missing header comment. All non trivial methods should have javadoc format header comments.